invenio_rdm_records/resources/config.py

```diff
@@ -110,7 +110,7 @@ class RDMRecordResourceConfig(RecordResourceConfig, ConfiguratorMixin):
     routes["restore-record"] = "/<pid_value>/restore"
     routes["set-record-quota"] = "/<pid_value>/quota"
     routes["set-user-quota"] = "/users/<pid_value>/quota"
-    routes["list-all"] = "/all"
+    routes["list-all"] = "/administration/records"
```

Comment:
changed to dedicated endpoint, I will try to solve it in a better way in a separate PR